ext/search_kit/ang/crmSearchAdmin/displays/colType/buttons.html

```diff
@@ -1,4 +1,11 @@
 <div class="form-inline">
+  <label for="crm-search-admin-hide-label--{{$index}}">
+    {{:: ts('Hide header field')}}
```

Comment:
This sort of option already exists for other types of displays, e.g. List:

![image](https://github.com/user-attachments/assets/51297115-e3b7-448b-a631-7b6fa6da3295)

So maybe we could generalize that?